src/P503/AMD64/fp_x64_asm.S

```diff
@@ -26,6 +26,8 @@
 #define p503x2_6   0xC08B8D7BB4EF49A0
 #define p503x2_7   0x0080CDEA83023C3C
 
+.text

```

Comment:
shouldn't this be rather ``.section .rodata`` ?

Reply:
.text is also a read-only section.
As p503p1_nz is accessed using the instruction pointer (rip+p503p1_nz) to allow relocation at link when building position independent code, it seemed safer to me to set its definition in the code segment but it shall also work in a .rodata section which is relocated I think in the same way the code section is.

Reply:
on some systems there is no read access for ``.text`` - it's true for new android

Reply:
Ah ok, it seems anyway that the position of the data segment relatively to the code segment is fixed during relocation, so yes, that .text shall be set after the p503p1_nz declaration and a .section .rodata set instead.

A similar edition shall be done in the p751 assembly code.

Are you reviewing this pull request to actually merge it in the repository? I thought it was more or less not maintained anymore, if not I may have more code to submit, mostly some cmake build scripts.

Reply:
@jeannotlapin it's just side comment

Reply:
Hi guys, big thanks for the interest and suggestions. I'm preparing a major update to the library, which already addresses some of the issues. It's coming soon (around beginning April) so I ask you to hold any further suggestions for now.